examples/deepcopy-gen/generators/deepcopy.go

```diff
@@ -523,11 +523,12 @@ func (g *genDeepCopy) doMap(t *types.Type, sw *generator.SnippetWriter) {
 				sw.Do("}\n", nil)
 				sw.Do("(*out)[key] = *newVal\n", nil)
 			} else {
-				sw.Do("if newVal, err := c.DeepCopy(&val); err != nil {\n", nil)
+				sw.Do("newVal, err := c.DeepCopy(&val)\n", nil)
```

Comment:
lint is a moving target, and (as in this case) sometimes encourages non-idiomatic code. I don't think jumping through hoops to silence it is a good i dea.